nixos/modules/services/databases/riak-cs.nix

```diff
@@ -110,13 +118,6 @@ in
         '';
       };
 
-      extraAdvancedConfig = mkOption {
```

Comment:
Furthermore, `advanced.conf` can be a liability to breaking the package functionality: a malformed `advanced.conf` will prevent the package from booting.